src/scss/4-pages/contact_us.scss

```diff
@@ -0,0 +1,31 @@
+.contact-form label {
+    font-size: 25px;
+    font-weight: 400px;
+    color: rgb(204, 204, 204);
+}
+
+.contact-form input {
+    width: 100%;
+    height: 60px;
+}
+
+.contact-form input::placeholder {
+    font-size: 20px;
+}
+
+.contact-form textarea {
+    min-height: 200px;
+}
+
+.contact-form textarea::placeholder {
+    font-size: 20px;
+}
+
+.contact-form-submit {
+    margin-top: 20px;
+    color: #ffffff;
+}
```

Comment:
This is a sass file. We should be seeing css of the form
```
.contact-form {
    input {
        width: 100%;
        &::placeholder {
            font-size: 20px;
         }
   textarea {
       min-height: 200px;
   }
}; 
```
Refer to http://sass-lang.com/styleguide/code